### DISCUSSION TITLE
I can't access a number of my repositories

### DISCUSSION THREAD

**Discussion by bwellington** [community/community#43483] [Repositories] [answered]
Many of my repositories are inaccessible. They display the message, "This repository is moving to a newer, better home. It’ll be back in just a few minutes." How do I resolve this? They have been down for at least 24 hours at this point.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @bwellington ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. 

That being said, I'd love to try and help, have you tried accessing the repositories on a different browser?

**Comment by bwellington** [comment]
Hi, thank you for moving my question to the appropriate category. Yes, the same repositories are inaccessible in different browsers.

**Comment by bwellington** [comment, marked as answer]
I was able to restore my repo by changing its visibility to public and then back to private.
